docs/users/smirnoff.md

```diff
@@ -1,2 +1,76 @@
-:::{include} ../../The-SMIRNOFF-force-field-format.md
+# SMIRNOFF (SMIRks Native Open Force Field)
+
+:::{hint} 
+The actual text of the SMIRNOFF specification has been moved to the
+[standards repository].
 :::
+
+OpenFF releases all its force fields in SMIRNOFF format. SMIRNOFF is a format
+developed by OpenFF; its specification can be found in our
```

Comment:
(blocking) This is probably what I'd asked for, but now that I see it, it seems easy for someone to miss the links to the full spec and think that this summary page IS the spec, or slow down people who want to get to the full thing. So I think this would be better if there were two headers, along the lines of:
* The link to the full spec
* A user-friendly summary of the SMIRNOFF format

(really there's a lot of latitude in what the exact headers say, I'm more interested in people getting the impression that there are two important artifacts represented on this page: the summary, and the link to the full spec)